### ISSUE TITLE
Multiple removed Techniques still live on 2.1 and 2.2 sites

### ISSUE THREAD

**Issue by fstrr** [w3c/wcag#3651]
Back in May 2022, I created an [issue to remove out of date Techniques](https://github.com/w3c/wcag/issues/2482). Revisiting that now, some of the Techniques have been removed from the repo but are still on the WCAG21 and WCAG22 versions of the site.

* [H4 is live on the site](https://www.w3.org/WAI/WCAG22/Techniques/html/H4) but [isn’t in the repo](https://github.com/w3c/wcag/tree/main/techniques/html).
* [H73 is live on the site](https://www.w3.org/WAI/WCAG22/Techniques/html/H73) but [isn’t in the repo](https://github.com/w3c/wcag/tree/main/techniques/html).
* [H35 is live on the site](https://www.w3.org/WAI/WCAG22/Techniques/html/H35) but [not in the repo](https://github.com/w3c/wcag/tree/main/techniques/html).
* [H45 is live on the site](https://www.w3.org/WAI/WCAG22/Techniques/html/H45.html) but [not in the repo](https://github.com/w3c/wcag/tree/main/techniques/html).
* [H46 is live on the site](https://www.w3.org/WAI/WCAG22/Techniques/html/H46.html) but [not in the repo](https://github.com/w3c/wcag/tree/main/techniques/html).
* [H60 is live on the site](https://www.w3.org/WAI/WCAG22/Techniques/html/H60) but [not in the repo](https://github.com/w3c/wcag/tree/main/techniques/html).

Side note: I'm in the process of creating separate issues for the remaining Techniques in the 2022 issue so it's more manageable.

**Comment by iadawn** [comment]
Looking into this there are two types of no longer relevant Techniques, in as much as there are no Success Criteria that reference the Technique:

* Ones which have been removed from Github
* Ones which haven't

H4 is an example of the former and [G136](https://www.w3.org/WAI/WCAG22/Techniques/general/G136) of the latter.

W3C has a general principle for persistence in URIs so I am hesitant to remove these completely. However, it would be good to include some text to explain that they are no longer relevant and, possibly, why. Currently they all have the text:

"This technique is not referenced from any Understanding document."

In the "about this technique" section.

I would propose coming up with some text that is a bit more than just "nothing points to this" to capture that it has been specifically removed.

Thoughts anyone?

**Comment by rscano** [comment]
"This technique is not referenced from any Understanding document for this wcag version".

For e.g. 
https://www.w3.org/WAI/WCAG22/Techniques/html/H60 (note header said wcag 2.1)

For wcag 2.0 is valid but not since 2.1.

https://www.w3.org/WAI/WCAG20/Techniques/html/H60

**Comment by fstrr** [comment]
Definitely agree that there should be some text that explains that they're no longer relevant any why. Just saying a Technique isn't linked to anything isn't helpful in explaining that a technique shouldn't be used.

**Comment by giacomo-petri** [comment]
Resolved via https://github.com/w3c/wcag/pull/3975

Closing as completed; please reopen if further concerns arise.
